projects/packages/my-jetpack/_inc/components/connected-product-card/index.jsx

```diff
@@ -62,9 +69,15 @@ const ConnectedProductCard = ( { admin, slug, children, showMenu = false, menuIt
 	] );
 
 	const handleInstallStandalone = useCallback( () => {
-		installStandalonePlugin().then( () => {
-			window?.location?.reload();
-		} );
+		setInstallingStandalone( true );
+
+		installStandalonePlugin()
+			.then( () => {
+				window?.location?.reload();
+			} )
```

Comment:
Minor suggestion. Not tested.
```suggestion
			.then( window?.location?.reload )
```

Reply:
Did not work